06.api-reference/28.picks/docs.md

```diff
@@ -106,16 +104,16 @@ An intersection result for a Ray Pick.
 
 ## Methods <a id="methods"></a>
 
-### `createPick(type, properties)`<a id="m1"></a>
+### `createPick(type, properties)`<a id="createPick"></a>
 
 Adds a new top-level menu. 
```

Comment:
A little confused by what is meant by "Adds a new top-level menu"

Reply:
This was a typo from the previous author.